### ISSUE TITLE
To increase tap target size for navbar icons

### ISSUE THREAD

**Issue by koderjoker** [internetarchive/iaux#27]
**Is your feature request related to a problem? Please describe.**
The following issue aims at making the navbar more accessible.

[Material design accessibility guidelines](https://material.io/design/usability/accessibility.html) state touch targets should be 48dp by 48dp with a 8dp separation size.

Current tap targets sizes for mobile are (px): 
icons on the left < 25 x 30
IA’s logo 40 x 30
Hamburger dropdown 41 x 30 
Search dropdown 62 x 30 
Upload icon 22.5 x 30
User details icon 25 x 30

![Screen Shot 2019-03-25 at 7 07 22 PM](https://user-images.githubusercontent.com/30471843/54923941-49b37400-4f31-11e9-9681-e52848fa3a97.png)

**Describe the solution you'd like**
To increase tap target sizes, even if it means removing some items from the navbar. Also refer to issue #28, #29

**Comment by pezvi** [comment]
Thanks for documenting! Claiming this one as an internal issue. :)
